app/task_routes.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, request, jsonify, abort, make_response
+from app import db
+from app.models.task import Task
+from app.routes_helper import validate_item_by_id
+from datetime import datetime
+import os
+import requests
```

Comment:
Teeny, tiny nit: it's standard practice to import external libraries first then import classes / functions from our own codebase. 

Kudos points for: each of those import sections to also be alphabetized!